### ISSUE TITLE
Chat | Placeholders disappear

### ISSUE THREAD

**Issue by kainlighty** [Zrips/CMI#8248]
### Description of Issue

When using the plugin https://github.com/moom0o/AnarchyExploitFixes and when using CMI Chat, placeholders disappear. They are replaced with emptiness, I disabled everything related to the chat in that plugin and it didn't help. Can you find a conflict?

### Version Information

```shell
--------------------------------------------------
Version 9.6.8.0 Unknown MySQL
CMILib: 1.4.3.0
Server: Purpur(1171) 1.16.5-R0.1-SNAPSHOT-
CMI economy: Enabled Vault: 1.7.3-CMI(CMIEInjector) CMI Chat: Enabled
Modules -> 40 enabled 19 disabled: spawnerProximity, cuffed, hpBossBar, customMessages, tablist, afk, skin, holograms, anvilRenameColor, jail, spawners, kits, bossBarCompass, deathMessages, ranks, firstJoinMessages, playerCombat, votifier, spawnerCharge
--------------------------------------------------
```

### Errors

```shell
-
```

### Relevant Config Sections

```yaml
Chat.yml
```

### Relevant Plugins

AnarchyExploitFixes 2.5.2

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by kainlighty** [comment]
I will add: Everything works with other chat plugins

**Comment by kainlighty** [comment]
Also, Oraxen does not want to be friends with prefixes from pictures, although it accepts them in the chat (only messages written by the player).

**Comment by kainlighty** [comment]
![image](https://github.com/Zrips/CMI/assets/111251772/64d9be15-022e-4026-b982-04e66414e954)

**Comment by kainlighty** [comment]
Without AEF plugin everything works

**Comment by kainlighty** [comment]
![image](https://github.com/Zrips/CMI/assets/111251772/c9b577c2-17c8-4e45-b487-004129f56d2d)
I don't know after what, but apparently after updating the plugin (because I didn't change anything in AEF and didn't touch any configs at all). At the moment, the problem is gone.

**Comment by kainlighty** [comment]
The problem has reappeared, I don't understand why. I don't see such problems in chatty, as an example

**Comment by Zrips** [comment]
So to be clear here, there is an issue for you when using placeholders like %luckperms_prefix% but its fine with base {prefix} ones? 
As it worked for you for some time, I could assume that there is some issue with plugins load order.
Could you provide config of your AEF plugin?

**Comment by kainlighty** [comment]
> So to be clear here, there is an issue for you when using placeholders like %luckperms_prefix% but its fine with base {prefix} ones? As it worked for you for some time, I could assume that there is some issue with plugins load order. Could you provide config of your AEF plugin?

There is no point in setting it up, since the standard one also has this problem, and no matter how I try to set it up, nothing comes out.

**Comment by kainlighty** [comment]
> Tried to debug it with AEF default config file and most of the protections enabled, loaded before and after CMI, in every case it just worked and provided correct placeholder values. Do you have a chance to test this issue on a clean server with CMI and AEF plugin just to be sure that it actually conflicts with this particular plugin?

I have already tested this on a server running on VDS. The result is always the same. There is a plugin -> no placeholders, no plugin (aef) -> there are placeholders. I can try to insert CMI in depend, maybe this will affect the order of loading and help. I will inform you about it.

**Comment by kainlighty** [comment]
Yes. I've done a couple more tests and there is such a problem when loading it. Despite the fact that I sometimes noticed in the chat that the placeholders worked, but I didn't understand how this was happening. I'm confused

**Comment by Zrips** [comment]
So changing load order fixed issue in your case? I still reproduce this issue. Everything looks to be just working. Something specific with your entire server setup or some configurations is happening. Its definitely a confusing case.

**Comment by kainlighty** [comment]
Unfortunately, it does not work, it has been tested on different versions of kernels (which means on different servers). Maybe there is something that can affect it in the configurations of your plugin? You have so many of them that if I install the standard one, then it will still not be clear what the problem is. They have the source code here, can you look at it and tell me in which direction to drip? I looked, but it didn't become clearer to me, since I didn't see the source code of your plugin.

**Comment by Zrips** [comment]
Well there isn't anyone else (as far as I know) with issue like yours, so it must be isolated to your specific setup with your specific config files. Everything seamingly works just fine for me, was running AEF on my test server for few days, haven't encountered any issues so far. 
Best test what you could do on your end is to just run everything with minimal plugins and with default config files for as many thing as you can, including server configurations. And check if that still produces same issue or not. Or you could send me your config files for atleast CMI and AEF so that I could have more close example to yours and hopefully I can reproduce this issue. At the moment I can, then it's relatively easy fix. You can get in contact with me over discord and send files over there if you don't want to publish those in public space.
https://discord.gg/hAGMmPP join server and you can send PM

**Comment by kainlighty** [comment]
I tried to run it on a test server, but there is no such error. I don't understand why (I would run with all plugins, but I'm afraid that some developers may block the license, as some wrote - the transfer is prohibited, and I don't know exactly how they will detect the transfer). I tried to run it on a test server, but there is no such error. I don't understand why (I would run with all plugins, but I'm afraid that some developers may block the license, as some wrote - the transfer is prohibited, and I don't know exactly how they will detect the transfer)
